### PR TITLE
Add unverified users docs

### PR DESCRIPTION
**Pull request by BobbyRadford**

### CODE REVIEW

File: configuration/authentication-methods/unverified.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<Snippet file="unverified-users-description.mdx" />
+
+## Enabling Unverified Users
+Enabling Unverified Users is easy and does not require any additional code. Simply follow the steps below to enable Unverified Users for your application.
+
+1. Open your app in the [Rownd platform](https://app.rownd.io/applications). Then click on the **Sign-in methods** tab on the left to adjust your flow.
+
+   ![Finding authentication methods is easy; just select "Sign-in methods" from the nav pane.](/images/sign-in-methods-page.png)
+
+2. Scroll down to the bottom of the page and, and press the toggle to next to **Unverified users**.
```
Comment (mhamann) [inline configuration/authentication-methods/unverified.mdx:16]: Then press **Save**

File: _snippets/unverified-users-description.mdx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Rownd's Unverified Users feature allows new users to sign in with their email or phone number without immediately needing to verify. Existing users, or sign-in attempts with an email or phone number for which a user already exists will be required to verify immediately.
```
[the remark this answers is not in the document]
Comment (BobbyRadford) [reply _snippets/unverified-users-description.mdx:1]: Yeah good point

File: configuration/authentication-methods/unverified.mdx (hunk 1 above)
Comment (BobbyRadford) [reply configuration/authentication-methods/unverified.mdx:16]: Good point. Also an extra "and," there

Comment (BobbyRadford) [reply configuration/authentication-methods/unverified.mdx:16]: Actually that whole wording is messed up.

**Review by mhamann — APPROVED** [review]

**Pull request merged**